packages/rudy/src/middleware/call/utils/shouldCall.js

```diff
@@ -1,16 +1,14 @@
 // @flow
-import type { LocationState } from '../../../flow-types'
 import { isServer } from '@respond-framework/utils'
+import type { LocationState } from '../../../flow-types'
 import { isHydrate } from '../../../utils'
 
-export default (name, route, req) => {
+export default (name, route, req, { runOnServer, runOnHydrate }) => {
```

Comment:
Do you think its a good idea to make the default behaviour that it runs on the server and not hydrate? Seems like usually people want callbacks to happen at least at some point on initial load.

Reply:
I can't really visualize the use-case for these from the end-users POV, as I haven't used SSR personally. However I do think that skipOnServer (+ reversing logic in shouldCall) is a better API than runOnServer = true in that case.

Reply:
I like that both options have the same positive polarity (`runOnX` + `skipOnY` I think is more confusing) but potentially one could default to true?

Reply:
I agree that it's confusing, but implicit true seems even worse. :-)
Undefined is false in JS so it's easy to see when an option is not set. If we change that, it suddenly becomes hard to know whether required to say "fooFlag: false" or "barFlag: true", and it somehow becomes a tristate.
Is there a third option?

Reply:
I forget which babel plugin its in, but I think I've seen code like

```javascript
(name, route, req, { runOnServer = true, runOnHydrate = false }) => {}
```

Reply:
Don't worry though, I think its fine the way it is, simple and clear.